support-us.md

```diff
@@ -0,0 +1,33 @@
+---
+layout: page
+order: 4
+title: "Support Us"
+titleDisplay: "Support Us"
+redirect_from:
+  - "/2018/swag/"
+  - "/swag/"
+---
+
+<div class="row">
+  <div class="six columns">
+    <figure>
+      <img src="/images/swag/pins-stickers.png" alt="Buttons and stickers from 2018 Our Networks">
+      <figcaption><p>Our Networks buttons and stickers.</p></figcaption>
+    </figure>
+  </div>
+  <div class="six columns">
+    <figure>
+      <img src="/images/swag/tote-design.png" alt="Front of 2018 Our Networks Tote">
+      <figcaption><p>Printed front and back with shoutouts to our wonderful sponsors.</p></figcaption>
```

Comment:
```suggestion
      <figcaption><p>Totes printed front and back with shoutouts to our wonderful sponsors.</p></figcaption>
```

Reply:
this agrees with how we refer to other swag (plural--buttons and stickers) and makes clear it is a tote!